Created popup form and "contact us" page

rybakma1 opened the pull request:
Создал popup форму на странице "index.html" и страницу "contact us.html"

rybakma1 committed:
Created a popup form in index.html

Разметил popup форму на странице index.html.
Создал страницу contact us и скопировал в нее хедер, футер и попап форму для дальнейшей доработки.

rybakma1 committed:
Created by Contact-Us.html

Сверстал страницу Contact-Us.html

FedchenkoVyacheslav commented on contact-us.html:
```diff
@@ -0,0 +1,113 @@
+    <section>
+        <h2 class="popup-form visually-hidden">Popup form</h2>
+        <form id="popup" name="user-message" method="POST">
+            <fieldset>
```
Нужны классы всем элементам формы

FedchenkoVyacheslav commented on contact-us.html:
```diff
@@ -0,0 +1,113 @@
+            <li class="footer-bottom-item"><p class="footer-and-text">Web Design — Direct Line Development</p></li>
+        </ul>
+    </footer>
+    
```
Форма так же должна быть после footer

rybakma1 committed:
added classes to forms and fixed grammar

Добавил классы ко всем элементам форм, добавил адрес сервера для отправки данных с форм. Исправил некоторые грамматические ошибки.
